config/credentials.yml.enc

```diff
@@ -1 +1 @@
-Cu8bv6DTSY+mZ8KJPw6tqNeW9MRvc6fl+Wy6xNRemztwGCire8mu2sP58iP5PMoann47afI8Yxs8H+HvAD4wczCQwZ28ipy8PywrJWhqsWaKhA5iz4PgJUniKHAM0Xcrbfei2F2pUqDpipEmSjtNirXDjeQIMMQO8mUDhylWxk0D51s/V0Ia421t9v11kFBayeqQJh4/80cdur0f9RYctet1a3UPJrwsjVP4CkmYUlP+E2xddEY4JRI0hX4d/w9XqgO/vIFZeGH4W7pGRVyR6mB0cugJOWj2IyJldW82wYtRpbgN+Fs+PagHojOS1UoKu9e7qW8B8GJ5LjW7V3xQ1A/9zzMwRApwT2QrCbyGUW9uD4GO9pL0hKPwQcMPC4/eZ2Ye4Nr/EmvFEfczhyKpsMnkYH9tiSO8iPXzJI22stZaiBRUbaeuoL8M1DeRKxlr6rR81t95uRC6bMBXqoG25S3k3OAGET6XGSOJzkWOT61LVd/TqZamIPMlRVABzlkzHaSPJb6HmAgaSsK0nfnoWVwtLvagwiaA6hKTeFHQFj0uESd50Qt19P09WnN0daLUsA==--kXdPerAvKX9BNqcO--oBpfONbfQTSKGMIe526QIw==
\ No newline at end of file
+Cu8bv6DTSY+mZ8KJPw6tqNeW9MRvc6fl+Wy6xNRemztwGCire8mu2sP58iP5PMoann47afI8Yxs8H+HvAD4wczCQwZ28ipy8PywrJWhqsWaKhA5iz4PgJUniKHAM0Xcrbfei2F2pUqDpipEmSjtNirXDjeQIMMQO8mUDhylWxk0D51s/V0Ia421t9v11kFBayeqQJh4/80cdur0f9RYctet1a3UPJrwsjVP4CkmYUlP+E2xddEY4JRI0hX4d/w9XqgO/vIFZeGH4W7pGRVyR6mB0cugJOWj2IyJldW82wYtRpbgN+Fs+PagHojOS1UoKu9e7qW8B8GJ5LjW7V3xQ1A/9zzMwRApwT2QrCbyGUW9uD4GO9pL0hKPwQcMPC4/eZ2Ye4Nr/EmvFEfczhyKpsMnkYH9tiSO8iPXzJI22stZaiBRUbaeuoL8M1DeRKxlr6rR81t95uRC6bMBXqoG25S3k3OAGET6XGSOJzkWOT61LVd/TqZamIPMlRVABzlkzHaSPJb6HmAgaSsK0nfnoWVwtLvagwiaA6hKTeFHQFj0uESd50Qt19P09WnN0daLUsA==--kXdPerAvKX9BNqcO--oBpfONbfQTSKGMIe526QIw==
```

Comment:
can you remove  this from your PR

Reply:
Ok I believe I replaced it with what's currently in master, but I can't seem to remove this change overall, even after force pushing

Reply:
Woah wtf idk where it went